server/tests/backends/sql_translator/test_snowflake_translator_steps.py

```diff
@@ -91,62 +94,131 @@ def snowflake_query_executor(domain: str, query_string: str = None) -> Union[Tes
         return TestDataSlice(df=pd.DataFrame(res))
 
 
+def _drop_table(table_array: list):
+    global SNOWFLAKE_TABLES_TESTS
+
+    """
+    This method will drop tables
+    As given parameter, we have list of tables we want to drop
+
+    """
+    print("[x] Cleaning tables in progress, please wait...")
```

Comment:
Thanks for this logging effort ! 
Can you use a logger instead of a print :pray:

Reply:
Updated !